server/routes/auth.js

```diff
@@ -0,0 +1,13 @@
+import Router from 'express';
+import AuthController from '../controllers/authController';
+import bnLog from '../helpers/log.util';
+import validateAuth from '../middlewares/validateAuth';
+// eslint-disable-next-line no-unused-vars
+import passportSet from '../config/passport';
+
+const router = Router();
+
+router
+  .post('/signin', validateAuth.validateSignin, bnLog(AuthController.signIn));
```

Comment:
Capitalize the last i as well: **validateSignIn**.

Reply:
Feedback implemented... Well renamed!